ocfweb/docs/templates/docs/partials/account.md

```diff
@@ -0,0 +1,37 @@
+What Your OCF Account Gets You
+
+Many think of the Open Computing Facility as an on-campus resource for free printing. While this is true, and printing is certainly our most-demanded service (we print hundreds of thousands of pages per year), there are plenty of other great services provided by the OCF, which come free with your account!
+
+## Free Computing
+
+The OCF was founded to provide all members of the Cal campus community with computing resources. Anytime we're open, all members of the community are welcome to use the computers for any reason – not just to print. All of the computers in the OCF run Linux as the operating system and have a variety of useful (although potentially unfamiliar) applications installed. Because of our committment to free, open-source software, we do not provide access to application suites such as Microsoft Office or Adobe Creative Cloud; instead, we provide free alternatives.
+* Google Chrome, Firefox, and Brave are all installed, allowing users to choose their preferred web browser.
+* The LibreOffice suite is installed as an alternative to Microsoft Office. It is mostly cross-compatible with Office files, meaning one can import or export .docx/.pptx/.xlsx files. Users can also use Google Docs from a web browser.
+* GIMP (GNU Image Manipulation Program) is provided as an alternative to Adobe Photoshop, and Inkscape replaces Adobe Illustrator. For animation and 3D modeling, we provide access to Blender.
+* Rather than iTunes/QuickTime or Windows Media Player, users can access music or video files using VLC.
+* The pre-installed text editor is Atom, and the pre-installed IDE is Visual Studio Code.
+* The OCF does not currently provide a true replacement for Adobe Acrobat; PDFs can be opened with the Document Viewer and edited using other applications.
+Most of these applications can be found on the desktop, and all applications can be accessed from the menu in the top-left corner of the screen, analogous to the Start menu in Windows.
+
```

Comment:
I would mention our scanners can be used for copying, and we have one computer with a disk reader.

Reply:
I think those services might be worth mentioning too.